iceoryx2-cli/iox2/src/commands.rs

```diff
@@ -0,0 +1,160 @@
+use colored::*;
+use std::env;
+use std::fs;
+use std::path::PathBuf;
+use std::process::{Command, Stdio};
+use thiserror::Error;
+
+#[derive(Clone, Debug)]
+struct CommandInfo {
+    name: String,
+    path: PathBuf,
+    is_development: bool,
+}
+
+#[derive(Error, Debug)]
+pub enum ExecutionError {
+    #[error("Command not found: {0}")]
+    NotFound(String),
+    #[error("Execution failed: {0}")]
+    Failed(String),
+}
+
+pub fn list() {
+    println!("Installed Commands:");
+    let installed_commands = find();
+    for command in installed_commands {
+        println!(
+            "  {}",
+            format!(
+                "{}{}",
+                if command.is_development { "(dev) " } else { "" },
+                command.name.bold(),
+            )
+        );
+    }
```

Comment:
Can you sort the output before printing it?

Also, if I get this right it would look like this
```
(dev) pub
pub
```
When sorted, it would maybe make more sense to have it like this
```
pub
pub (dev)
```

Reply:
@elBoberido Fixed.